conda/activate.py

```diff
@@ -788,84 +816,87 @@ def path_identity(paths):
 
 
 class PosixActivator(_Activator):
-
     def __init__(self, arguments=None):
-        self.pathsep_join = ':'.join
-        self.sep = '/'
+        self.pathsep_join = ":".join
+        self.sep = "/"
         self.path_conversion = native_path_to_unix
-        self.script_extension = '.sh'
+        self.script_extension = ".sh"
         self.tempfile_extension = None  # write instructions to stdout rather than a temp file
-        self.command_join = '\n'
+        self.command_join = "\n"
 
-        self.unset_var_tmpl = 'unset %s'
+        self.unset_var_tmpl = "unset %s"
         self.export_var_tmpl = "export %s='%s'"
         self.set_var_tmpl = "%s='%s'"
         self.run_script_tmpl = '. "%s"'
 
-        self.hook_source_path = join(CONDA_PACKAGE_ROOT, 'shell', 'etc', 'profile.d', 'conda.sh')
+        self.hook_source_path = join(CONDA_PACKAGE_ROOT, "shell", "etc", "profile.d", "conda.sh")
 
         super().__init__(arguments)
 
     def _update_prompt(self, set_vars, conda_prompt_modifier):
-        ps1 = self.environ.get('PS1', '')
-        if 'POWERLINE_COMMAND' in ps1:
+        ps1 = self.environ.get("PS1", "")
+        if "POWERLINE_COMMAND" in ps1:
             # Defer to powerline (https://github.com/powerline/powerline) if it's in use.
             return
-        current_prompt_modifier = self.environ.get('CONDA_PROMPT_MODIFIER')
+        current_prompt_modifier = self.environ.get("CONDA_PROMPT_MODIFIER")
         if current_prompt_modifier:
-            ps1 = re.sub(re.escape(current_prompt_modifier), r'', ps1)
+            ps1 = re.sub(re.escape(current_prompt_modifier), r"", ps1)
         # Because we're using single-quotes to set shell variables, we need to handle the
         # proper escaping of single quotes that are already part of the string.
         # Best solution appears to be https://stackoverflow.com/a/1250279
         ps1 = ps1.replace("'", "'\"'\"'")
-        set_vars.update({
-            'PS1': conda_prompt_modifier + ps1,
-        })
+        set_vars.update(
+            {
+                "PS1": conda_prompt_modifier + ps1,
+            }
+        )
```

Comment:
```suggestion
        set_vars["PS1"] = conda_prompt_modifier + ps1
```

Reply:
I ran this file through black to silence the linter. I would rather avoid making other stylistic changes.